mithril-aggregator/src/artifact_builder/cardano_database_artifacts/digest.rs

```diff
@@ -87,28 +120,45 @@ impl DigestArtifactBuilder {
             }
         }
 
-        if locations.is_empty() {
-            return Err(anyhow!(
-                "Failed to upload digest archive with all uploaders"
-            ));
-        }
+        locations.push(self.aggregator_location()?);
 
         Ok(locations)
     }
+
+    fn aggregator_location(&self) -> StdResult<DigestLocation> {
```

Comment:
Maybe this name would be clearer:
```suggestion
    fn aggregator_digests_route_location(&self) -> StdResult<DigestLocation> {
```

Reply:
Done